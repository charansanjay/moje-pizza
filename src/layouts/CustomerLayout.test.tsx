import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { describe, it, expect } from 'vitest';

import { CustomerLayout } from './CustomerLayout.tsx';

describe('CustomerLayout - LAYOUT', () => {
  it('should render the Outlet component', () => {
    const { container } = render(
      <MemoryRouter>
        <CustomerLayout />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});