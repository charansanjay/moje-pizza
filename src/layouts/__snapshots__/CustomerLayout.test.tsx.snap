// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CustomerLayout > should render the Outlet component 1`] = `<div />`;

exports[`CustomerLayout - LAYOUT > should render the Outlet component 1`] = `<div />`;
